packages/storage-ui/cypress/tests/bucket-management.cy.ts

```diff
@@ -144,5 +143,151 @@ describe("Bucket management", () => {
       bucketsPage.bucketItemName().eq(0).should("have.text", chainSafeBucketName)
       bucketsPage.bucketItemName().eq(1).should("have.text", ipfsBucketName)
     })
+
+    it("can rename a folder inside the ipfs bucket", () => {
+      const ipfsBucketName = `ipfs bucket ${Date.now()}`
+      const folderName = `folder ${Date.now()}`
+      const newFolderName = `new folder name ${Date.now()}`
+
+      cy.web3Login({ deleteFpsBuckets: true })
+      navigationMenu.bucketsNavButton().click()
+
+      // create a new bucket and go inside the bucket
+      bucketsPage.createBucket(ipfsBucketName, "ipfs")
+      bucketsPage.bucketItemRow().should("have.length", 1)
+      bucketsPage.bucketItemName().dblclick()
+
+      // create a folder inside the bucket
+      bucketContentsPage.createNewFolder(folderName)
+
+      // ensure an error is displayed if the edited name of the folder is blank
+      bucketContentsPage.renameFileOrFolder("{selectall}{del}")
+      bucketContentsPage.fileRenameErrorLabel().should("be.visible")
+
+      // rename a folder
+      bucketContentsPage.fileRenameInput().type(`${newFolderName}{enter}`)
+      bucketContentsPage.fileItemName().contains(newFolderName)
+
+      // ensure the original name of the folder persists if the rename submission is blank
+      bucketContentsPage.renameFileOrFolder("{selectall}{del}{esc}")
+      bucketContentsPage.fileRenameInput().should("not.exist")
+      bucketContentsPage.fileItemName().contains(newFolderName)
+
+      // ensure that the name of the folder is reset when renaming is canceled
+      bucketContentsPage.renameFileOrFolder("{selectall}abc{del}{esc}")
+      bucketContentsPage.fileRenameInput().should("not.exist")
+      bucketContentsPage.fileItemKebabButton().first().click()
```

Comment:
I don't think `first` is necessary as there is only one folder present?
```suggestion
      bucketContentsPage.fileItemKebabButton().click()
```

Reply:
I will fix this ones and the {selectall}{del} error in Files as well